source/views/streaming/streams/list.js

```diff
@@ -128,8 +138,8 @@ export class StreamListView extends React.PureComponent {
         ItemSeparatorComponent={ListSeparator}
         keyExtractor={this.keyExtractor}
         style={styles.listContainer}
-        refreshing={this.props.loading}
```

Comment:
Where are the props to this component defined? I don't see them, and it's hampering my ability to understand this component… 😜

Reply:
This removes a reference to `props` that are not used. This PR fixes anything relating to `props`.